test/unit/profiles/profile_test.rb

```diff
@@ -99,6 +99,15 @@
     end
   end
 
+  describe 'skips loading on unsupported platform' do
+    let(:profile_id) { 'windows-only' }
```

Comment:
I think we should add a test reflecting @frezbo's concern - a run on a linux platform should not skip a profile that supports windows and linux.  That may already be covered in another test - if so, link it and I'll be quiet :)

Reply:
I'll add a example profile to our mocks that supports bsd, linux, and windows.